core/lib/libtomcrypt/rsa.c

```diff
@@ -356,6 +366,16 @@ TEE_Result sw_crypto_acipher_rsaes_decrypt(uint32_t algo,
 		EMSG("tee_algo_to_ltc_hashindex() returned %d", (int)res);
 		goto out;
 	}
+	if (algo != TEE_ALG_RSAES_PKCS1_V1_5) {
+		res = tee_algo_to_ltc_hashindex(mgf_algo, &ltc_mgfindex);
+		if (res != TEE_SUCCESS) {
+			EMSG("tee_algo_to_ltc_hashindex() returned %d for mgf algo %#"PRIx32,
+				(int)res, mgf_algo);
```

Comment:
`res` should also be printed as a PRIx32.

I see now that this is copied from above, but two wrongs don't make one right.

Reply:
Ack, also changed the message above since I'm here.